python-sdk/src/astro/databases/__init__.py

```diff
@@ -37,5 +40,7 @@ def create_database(conn_id: str, table: BaseTable | None = None) -> BaseDatabas
     module_path = CONN_TYPE_TO_MODULE_PATH[conn_type]
     module = importlib.import_module(module_path)
     class_name = get_class_name(module_ref=module, suffix="Database")
-    database: BaseDatabase = getattr(module, class_name)(conn_id, table)
+    database: BaseDatabase = getattr(module, class_name)(
+        conn_id, table, load_options=load_options_list and load_options_list.get(getattr(module, class_name))
+    )
```

Comment:
I was wondering if we'd be able to simplify the code using something like this:
```suggestion
    database_class = getattr(module, class_name)
    load_options_list = class_to_load_options.get(database_class, load_options_list)
    database: BaseDatabase = database_class(
        conn_id, table, load_options=)
    )
```

Reply:
Added in a septate commit.